challenge_1/go/tml/src/strrev.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "bufio"
+)
+
+func strrev(s string) string {
+    r := []rune(s)
+    rs := []rune{}
```

Comment:
Utilize make() to create the new rune slice with enough space to hold the string being reversed.

rs := make([]rune, 0, len(s))